src/site/stages/build/drupal/graphql/landingPage.graphql.js

```diff
@@ -58,3 +66,43 @@ module.exports = `
     ${ADMIN}
   }
 `;
+
+let regString = '';
+queryParamToBeChanged.forEach(param => {
+  regString += `${param}|`;
+});
+
+const regex = new RegExp(`${regString}`, 'g');
+const landingPageFragmentModified = landingPageFragment.replace(
+  regex,
+  updateQueryString,
+);
+
```

Comment:
I do not understand why we have these string manipulations versus just using the correct value in the actual query, but without it I encountered a GraphQL error so from that I discovered how it is used. I assume there is some CMS-internal context/knowledge about this.